### PR TITLE
Add contains serialized data in requests flag to client messages [API-1687]

### PR DESCRIPTION
**Pull request by harunalpak**
Related issue: [#437](https://github.com/hazelcast/hazelcast-client-protocol/pull/437)

Add contains serialized data in requests flag to client messages for Typescript

### CODE REVIEW

**Review by mdumandag — DISMISSED** [review]
I am approving this PR but let's wait until the PR for it is merged to the Node.js client repo, before merging this.

**Comment by harunalpak** [comment]
> I am approving this PR but let's wait until the PR for it is merged to the Node.js client repo, before merging this.

Related nodejs PR merged. Could you approve, please?

**Review by mdumandag — APPROVED** [review]

**Review by srknzl — APPROVED** [review]

**Pull request merged**
